### PR TITLE
Create ME House Dems yamls

### PR DESCRIPTION
**Pull request by OdeclasV**

### CODE REVIEW

File: members/MEL000279.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+bioguide: MEL000279
+contact_form:
+  method: post
+  action:
+  steps:
+    - visit: "https://legislature.maine.gov/housedems/contactus/index.html"
+    - select:
+      - name: Representative
+        selector: select#select
+        value: Representative Heidi Brooks
```
Comment (tomdionesotes) [inline members/MEL000279.yaml]: `required: true`?

Comment (OdeclasV) [reply members/MEL000279.yaml]: question: is it necessary to add the `required` step even if the value (in this case the Rep's name) is hardcoded to the yaml?

Comment (tomdionesotes) [reply members/MEL000279.yaml]: iirc either simplebrowser or phantom skips non required fields either on the initial post or if there's a failure. We need this field present if we're going to submit to the correct rep, so it should be `required: true` IMO

Comment (OdeclasV) [reply members/MEL000279.yaml]: ohh good to know. Just added `required: true` for both Representative and Topic

File: members/MEL000279.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        selector: input#name
+        value: $NAME_FULL
+        required: true
+      - name: Address
+        selector: input#address
+        value: $ADDRESS_STREET
+        required: true
+      - name: Address2
+        selector: input#address2
+        value: $ADDRESS_STREET
```
Comment (tomdionesotes) [inline members/MEL000279.yaml]: should be street2 and required:false

Comment (OdeclasV) [reply members/MEL000279.yaml]: good catch!

**Pull request merged**